app/templates/base-styles/colors.hbs

```diff
@@ -79,10 +75,10 @@
 <section>
 	<h2 id="alerts">Alerts</h2>
 	<div class="colors-wrapper">
-		{{color-box hex='#ed1c24' name='Alert/Delete' variable='$wds-color-alert'}}
-		{{color-box hex='#f08a24' name='Warning/Reported' variable='$wds-color-warning'}}
-		{{color-box hex='#5ba200' name='Success/Approve' variable='$wds-color-success'}}
-		{{color-box hex='#1a5eb8' name='Message' variable='$wds-color-message'}}
+		{{color-box hex='#d71035' name='Alert/Delete' variable='$wds-color-alert'}}
```

Comment:
If you're updating those please update 
```scss
 // ## Alerts
 $wds-color-alert: #ed1c24;
 $wds-color-message: #1a5eb8;
 $wds-color-success: #5ba200;
 $wds-color-warning: #f08a24;
```
in `_colors.scss`. And replace hardcoded color hashed with fandom colors they're based on.